geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py

```diff
@@ -52,11 +52,12 @@
     "srid": "${srid}",
     "extent": "489246, 78873, 837119, 296543",
 }
-DIFF_NOTICE = "The changes visible on `a/CONST_create_template/<file>` should be done on `<file>`.\n" \
-    "An advise to be more effective: in most cases it concerns a file that you never customize, " \
-    "or a file that you have heavily customized, then respectively copy the new file from " \
-    "`CONST_create_template` (`cp CONST_create_template/<file> <file>`), respectively ignore " \
-    "the changes."
+DIFF_NOTICE = "You should apply the changes shown in the diff file on `a/CONST_create_template/<file>` " \
+    "on your project's `<file>`.\n" \
+    "Some advice to be more efficient: if the changes on a file concern a file that you never customize, " \
+    "you can simply copy the new file from `CONST_create_template` (`cp CONST_create_template/<file> <file>`)." \
```

Comment:
, and ad it in the `unmanaged_files` section of the `project.yaml.mako` file.

Reply:
I now added that, can you please check if I understood correctly?

Reply:
You steal needs to copy them, the changes is for the next upgrade...